tests/test_seeds.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pandas as pd
+import statsmodels.formula.api as sm
+from wildboottest.wildboottest import wildboottest
+import numpy as np
+
+@pytest.fixture
+def data():
+    N = 100
+    k = 2
+    G= 20
+    X = np.random.normal(0, 1, N * k).reshape((N,k))
+    beta = np.random.normal(0,1,k)
+    beta[0] = 0.1
+    u = np.random.normal(0,1,N)
+    y = 1 + X @ beta + u
+    cluster = np.random.choice(list(range(0,G)), N)
+    bootcluster = cluster
+    R = np.zeros(k)
+    R[0] = 1
+    B = 999
+    
+    return X, y, cluster, bootcluster, R, B
+
+def test_results_from_same_seed():
+    
+    df = pd.read_csv("https://raw.github.com/vincentarelbundock/Rdatasets/master/csv/sandwich/PetersenCL.csv")
+    df['treat'] = np.random.choice([0, 1], df.shape[0], True)
+    model = sm.ols(formula='y ~ treat', data=df)    
+    
+    for i in range(100):
+        a = wildboottest(model, param = "treat", cluster = df.year, B= 999, seed=11232198237961)
+        b = wildboottest(model, param = "treat", cluster = df.year, B= 999, seed=11232198237961)
+        pd.testing.assert_frame_equal(a,b)
```

Comment:
Could we add tests for the HC bootstrap here as well? As I have experienced some problems with numba's random seeds.

```
    for i in range(100):
        a = wildboottest(model, param = "treat", cluster = df.year, B= 999, seed=11232198237961)
        b = wildboottest(model, param = "treat", cluster = df.year, B= 999, seed=11232198237961)
        pd.testing.assert_frame_equal(a,b)
        c = wildboottest(model, param = "treat", B= 999, seed=11232198237961)
        d = wildboottest(model, param = "treat", B= 999, seed=11232198237961)
        pd.testing.assert_frame_equal(a,b)
```

Also, would something like

```
np.random.seed(1)
a = wildboottest(model, param = "treat", cluster = df.year, B= 999)
np.random.seed(1)
b = wildboottest(model, param = "treat", cluster = df.year, B= 999)
```
lead to identical inferences? If yes, maybe we can add a test there =)